asterix/ast_file.ksy

```diff
@@ -0,0 +1,43 @@
+meta:
+  id: ast_file
```

Comment:
the name is ambigous. AST can stand for "abstract syntax tree". Note that KS has no namespaces currently. Creating a new dir in tue root for such a niche application is strange. Probably `surveillance` dir should be created in the root and `asterix` should be moved there.

Reply:
I agree asterix probably is not to bee in root, but surveillance brings me CCTV and other things to my mind. Probably aeronautical or aero is a good name because there are other protocols in that field, even for radar data, you have also flight plan, etc.
About AST, in the class names it is a convenience name so you not write asterix, so it could be changed, no problem with that.